src/basho_bench_worker_sup.erl

```diff
@@ -71,6 +92,23 @@ worker_specs(0, Acc) ->
     Acc;
 worker_specs(Count, Acc) ->
     Id = list_to_atom(lists:concat(['basho_bench_worker_', Count])),
-    Spec = {Id, {basho_bench_worker, start_link, [Id, Count]},
-            transient, 5000, worker, [basho_bench_worker]},
+    %% Use "no_workers" atom for original non-worker case
+    Spec = {Id, {basho_bench_worker, start_link, [Id, {no_workers, Count, Count}, []]},
```

Comment:
I see, this refers to the `workers` config item? `no_workers` is still a bit confusing.